_sources/functions_in_cpp/mixed_up_code.rst

```diff
@@ -0,0 +1,344 @@
+Mixed Up Code Exercises
+-----------------------
+
+Answer the following **Mixed-Up Code** questions to practice what you have learned in this chapter.
```

Comment:
In the instructions, include these instructions: `Some lines contain **mistakes** or are **unnecessary** for the function - these lines should not be selected. Make sure to place the blocks at the right indentation levels!`

Reply:
(also include this in other mixed-up code files)